docs/patched_pdoc.py

```diff
@@ -73,22 +86,17 @@ def _add_to_inventory(dobj: pdoc_doc.Doc):
 
     pdoc_env.globals["add_to_inventory"] = _add_to_inventory
 
-else:
-    # Just dummy functions
-    def _empty(*args, **kwargs):
-        return ""
-
-    pdoc_env.globals["add_to_inventory"] = _empty
-
 # Run pdoc
-pdoc_env.globals["__hikari_version__"] = hikari.__version__
-pdoc_env.globals["__git_sha1__"] = hikari.__git_sha1__
+pdoc_env.globals["__hikari_version__"] = (
+    hikari.__git_sha1__.lower() if hikari.__git_sha1__.lower() in ("head", "master") else hikari.__version__
+)
 pdoc_main.cli()
 
 if generate_inventory:
     # Output the inventory
     text = project_inventory.data_file(contract=True)
     ztext = sphobjinv.compress(text)
```

Comment:
question, does this change fix/change the current behaviour where aliases for types/objects (e.g. `hikari.User` for `hikari.users.User`) aren't included in the generated objects.inv?

Reply:
Yes. `hikari.User` will no longer be a thing in the `object.inv`. I considered it a bug and fixed it as if it were